scene/resources/resource_format_text.cpp

```diff
@@ -554,7 +554,7 @@ Error ResourceLoaderText::load() {
 			}
 		}
 
-		MissingResource *missing_resource = nullptr;
+		Ref<MissingResource> missing_resource = nullptr;
```

Comment:
This is unnecessary.
```suggestion
		Ref<MissingResource> missing_resource;
```